pallets/parachain-staking/src/reward_config_calc.rs

```diff
@@ -66,8 +66,8 @@ impl<T: Config, R: RewardRateConfigTrait> CollatorDelegatorBlockRewardCalculator
 				Reward { owner: stake.id.clone(), amount: issue_number },
 			)
 		} else {
-			let collator_reward =
-				R::get_reward_rate_config().compute_collator_reward::<T>(issue_number);
+			let collator_reward = R::get_reward_rate_config()
```

Comment:
Oops, did I forget to remove this from the previous MR? Would you mind to check whether we can remove it or not?